tox/session.py

```diff
@@ -489,14 +490,21 @@ def setupenv(self, venv):
                         "not supported by virtualenv. Error details: %r" % e)
             except tox.exception.InvocationError as e:
                 status = (
-                        "Error creating virtualenv. Note that some special "
-                        "characters (e.g. ':' and unicode symbols) in paths are "
-                        "not supported by virtualenv. Error details: %r" % e)
+                    "Error creating virtualenv. Note that some special "
```

Comment:
this change seems unrelated here

Reply:
My IDE made that change - I'm reverting it back.